test/helpers/invoice-workflow.helper.ts

```
@@ -0,0 +1,288 @@
+  ): Promise<boolean> {
+    console.log(
+      `⏳ Waiting for task ${taskId} to reach status: ${expectedStatus}`,
+    );
+
+    const startTime = Date.now();
+
+    while (Date.now() - startTime < timeoutMs) {
+      try {
+        const task = await this.taskService.getTaskById(taskId);
+        if (task && task.status === expectedStatus) {
+          console.log(
```

ESLint

[error] Unexpected console statement.

(no-console)